src/js/views/globals/modal/modal_views.js

```diff
@@ -44,6 +44,7 @@ const SavingFooterView = View.extend({
     });
   },
   onBeforeDestroy() {
+    /* istanbul ignore if */
```

Comment:
I'm pretty sure we want to keep the protection on this. In scenarios where `this.animation` is `undefined`. When the modal view is destroyed before `onRender()` happens.